Dockerfile

```diff
@@ -15,4 +15,8 @@ RUN systemctl set-default multi-user.target
 
 COPY setup /sbin/
 
-CMD ["/sbin/init"]
+STOPSIGNAL SIGRTMIN+3
+VOLUME ["/sys/fs/cgroup"]
+
+# Workaround for docker/docker#27202
+CMD ["/bin/bash", "-c", "exec /sbin/init --log-target=journal 100>&1 1>&-"]
```

Comment:
Ah, thanks for the link to docker/docker#9212. Great find!

Could you change the number of the new FD to 3? It seems to me that 3 is a more common choice for "just a random new FD". 100 is so unusual and specific that I immediately thought there must be something special to it.

Also there doesn't seem to be a need to close FD 1, as systemd does that already (if it didn't, `/dev/console` would remain open and we wouldn't be seeing this issue), so I'd like to remove that part.

I tried changing `100>&1 1>&-` to `3>&1` and everything seemed to work just fine. If you could make that and the other changes I suggested, I'd be happy to merge the PR.